plugin/screenshot/screenshot.talon

```diff
@@ -1,3 +1,7 @@
+mode: command
```

Comment:
You don't need mode. `not tag: user.screenshot_disabled` is enough

I would probably also go `screenshot` in singular just because that is what we are already using.

Reply:
I made those changes. I agree with making my name consistent with the existing name.